server/utils/bedUtils.test.ts

```diff
@@ -71,57 +66,19 @@ describe('bedUtils', () => {
   })
 
   describe('actionCell', () => {
-    describe('when the user has the FUTURE_MANAGER role', () => {
-      const user = userDetailsFactory.build({ roles: ['future_manager'] })
-
-      it('returns a "V2" link to manage the room', () => {
-        expect(actionCell(bed, premisesId, user)).toEqual({
-          html: v2BedLink(bed, premisesId),
-        })
-      })
-    })
-
-    describe('when the user does NOT have the FUTURE_MANAGER role', () => {
-      const user = userDetailsFactory.build({ roles: ['manager'] })
-
-      it('returns a "V1" link to manage the room', () => {
-        expect(actionCell(bed, premisesId, user)).toEqual({
-          html: v1BedLink(bed, premisesId),
-        })
-      })
-    })
-
-    describe('when no user is given', () => {
-      const undefinedUser: UserDetails = undefined
-
-      it('returns a "V1" link to manage the room', () => {
-        expect(actionCell(bed, premisesId, undefinedUser)).toEqual({
-          html: v1BedLink(bed, premisesId),
-        })
+    it('returns a link to manage the room', () => {
+      expect(actionCell(bed, premisesId)).toEqual({
+        html: v2BedLink(bed, premisesId),
```

Comment:
In this case, the role has little effect, as the v1 version now produces a redirect to the v2 version, so most of these tests are redundant, and can be removed.